src/components/Calendar/__test__/calendar.spec.js

```diff
@@ -72,4 +72,30 @@ describe('Calendar', () => {
             }),
         );
     });
+    it('should fire onChange with a date as parameter when selectionType is single', () => {
+        const value = new Date('04/24/2019');
+        const mockChangeFn = jest.fn();
+        const simpleCalendar = mount(<Calendar value={value} onChange={mockChangeFn} />);
+        simpleCalendar
+            .find('table')
+            .find('button')
+            .at(7)
+            .simulate('click');
+        expect(mockChangeFn).toHaveBeenCalled();
+        expect(mockChangeFn).toHaveBeenCalledWith(new Date('04/08/2019'));
```

Comment:
here with:
```expect(mockChangeFn).toHaveBeenCalledWith(new Date('04/08/2019'));```
is suffcient since toHaveBeenCalledWith implies that the callback was called so remove:
```expect(mockChangeFn).toHaveBeenCalled();```